sim/signing/fake.go

```diff
@@ -84,3 +85,37 @@ func (s *FakeBackend) VerifyAggregate(payload, aggSig []byte, signers []gpbft.Pu
 	}
 	return nil
 }
+
+func (v *FakeBackend) MarshalPayloadForSigning(nn gpbft.NetworkName, p *gpbft.Payload) []byte {
```

Comment:
This skips _all_ hashing. We'll eventually hash with sha256 when signing, but only doing that once is much faster.